pom.xml

```diff
@@ -485,6 +486,8 @@
         <artifactId>maven-source-plugin</artifactId>
         <executions>
           <execution>
+            <phase>compile</phase>
```

Comment:
Why compile, this will be run on everysplit ATH.

Similar comment to below and the javadoc